truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -259,6 +260,7 @@ def create_tgi_build_dir(config: TrussConfig, build_dir: Path, truss_dir: Path):
         data_dir_exists=data_dir.exists(),
         credentials_exists=credentials_file.exists(),
         cached_files=cached_file_paths,
+        use_hf_secret=use_hf_secret,
```

Comment:
could you add a little bit more context, maybe in the PR description for this change?